src/qforte/system/molecular_info.py

```diff
@@ -162,3 +162,34 @@ def ccsd_energy(self):
     def ccsd_energy(self, ccsd_energy):
         self._ccsd_energy = ccsd_energy
 
+    @property
+    def point_group(self):
+        return self._point_group
+
+    @point_group.setter
+    def point_group(self, point_group):
+        self._point_group = point_group
+
+    @property
+    def orb_irreps(self):
+        return self._orb_irreps
+
+    @orb_irreps.setter
+    def orb_irreps(self, orb_irreps):
+        self._orb_irreps = orb_irreps
+
+    @property
+    def orb_irreps_to_int(self):
+        return self._orb_irreps_to_int
+
+    @orb_irreps_to_int.setter
+    def orb_irreps_to_int(self, orb_irreps_to_int):
+        self._orb_irreps_to_int = orb_irreps_to_int
+
+    @property
+    def hf_orbital_energies(self):
```

Comment:
Why exactly does `hf_orbital_energies` exist? Was this added for a feature that was scrapped from this PR?

Reply:
The hf_orbital_energies attribute is just a "placeholder" to facilitate a future PR. The intention is to extract the orbital energies directly from Psi4 instead of using QForte to calculate them as expectation values. Before we are able to do so, we need to deal with two issues.
1) We need to see how to extract orbital energies from OpenFermion (or get rid of OpenFermion altogether)
2) We need to write an HF code for model Hamiltonians to extract the orbital energies (or keep OpenFermion if such an option already exists).

Reply:
While _I'd_ prefer splitting `hf_orbital_energies` into a separate PR, because it's unrelated to "adding point group symmetry," fine.

I'm personally in favor of dropping OpenFermion support.

Reply:
I am also in favor of dropping OpenFermion. After this PR, I can work on removing the dependency.